dora/core/client/fs/src/test/java/alluxio/client/block/BlockStoreClientTest.java

```diff
@@ -1,535 +0,0 @@
-import static org.junit.Assert.assertThrows;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import alluxio.ClientContext;
-import alluxio.ConfigurationRule;
-import alluxio.annotation.dora.DoraTestTodoItem;
-import alluxio.client.WriteType;
-import alluxio.client.block.policy.BlockLocationPolicy;
-import alluxio.client.block.policy.options.GetWorkerOptions;
-import alluxio.client.block.stream.BlockInStream;
-import alluxio.client.block.stream.BlockOutStream;
-import alluxio.client.block.stream.BlockWorkerClient;
-import alluxio.client.block.stream.BlockWorkerDataReader;
-import alluxio.client.block.stream.GrpcDataReader;
-import alluxio.client.block.stream.NoopClosableResource;
-import alluxio.client.file.FileSystemContext;
-import alluxio.client.file.URIStatus;
-import alluxio.client.file.options.InStreamOptions;
-import alluxio.client.file.options.OutStreamOptions;
-import alluxio.conf.AlluxioConfiguration;
-import alluxio.conf.Configuration;
-import alluxio.conf.InstancedConfiguration;
-import alluxio.conf.PropertyKey;
-import alluxio.exception.ExceptionMessage;
-import alluxio.exception.PreconditionMessage;
-import alluxio.exception.status.UnavailableException;
-import alluxio.grpc.OpenFilePOptions;
-import alluxio.network.TieredIdentityFactory;
-import alluxio.resource.DummyCloseableResource;
-import alluxio.util.FileSystemOptionsUtils;
-import alluxio.util.network.NetworkAddressUtils;
-import alluxio.wire.BlockInfo;
-import alluxio.wire.BlockLocation;
-import alluxio.wire.FileBlockInfo;
-import alluxio.wire.FileInfo;
-import alluxio.wire.WorkerNetAddress;
-import alluxio.worker.block.BlockWorker;
-
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Sets;
-import io.grpc.stub.ClientCallStreamObserver;
-import io.grpc.stub.StreamObserver;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mockito;
-import org.mockito.stubbing.Answer;
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
-
-import java.io.Closeable;
-import java.util.AbstractMap;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.stream.Collectors;
-import javax.annotation.concurrent.ThreadSafe;
-
-/**
- * Tests for {@link BlockStoreClient}.
- */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({FileSystemContext.class})
-@DoraTestTodoItem(action = DoraTestTodoItem.Action.FIX, owner = "bowen",
-    comment = "recycle the applicable test cases after we remove BlockStore")
-public final class BlockStoreClientTest {
```

Comment:
This test i all based on BlockInStream/BlockOutStream, with a heavy dependency on `BlockLocationPolicy`. We do want to keep a few status-checking methods in `BlockStoreClient` and keep the `BlockStoreClient` class, but the tests in this file are no longer applicable. We should create a new test class for the remainder, after migrating/reimplementing the methods we keep.